Detectors/PHOS/reconstruction/src/Clusterer.cxx

```diff
@@ -113,23 +169,265 @@ void Clusterer::MakeClusters(const std::vector<Digit>* digits, std::vector<Clust
     } // loop over cluster
   }   // energy theshold
 }
+//__________________________________________________________________________
+void Clusterer::makeUnfoldings(const std::vector<Digit>* digits)
+{
+  //Split cluster if several local maxima are found
+
+  int* maxAt = new int[o2::phos::PHOSSimParams::Instance().mNLMMax]; // NLMMax:Maximal number of local maxima
```

Comment:
our coding guidelines disadvise using explicit new / delete, you better do 
``std::unique_ptr<int[]> maxAt = std::make_unique<int[]>(o2::phos::PHOSSimParams::Instance().mNLMMax)``
Then you don't need to delete it.

Reply:
As I pass it to other methods, use vector instead of unique_ptr